basiclibrary/src/main/java/basiclibrary/Maps_TallyElection.java

```diff
@@ -0,0 +1,61 @@
+package basiclibrary;
+import java.util.*;
+
+public class Maps_TallyElection {
+    public String weather(int[][] arr) {
+        int MAX = 0;
+        int MIN = arr[0][0];
+        String resultStr = "";
+        HashSet<Integer> uniqueTemp = new HashSet<Integer>();
+        for (int i = 0; i < arr.length; i++) {
+            for (int j = 0; j < arr[i].length; j++) {
+                uniqueTemp.add(arr[i][j]);
+                if (MAX < arr[i][j]) {
+                    MAX = arr[i][j];
+                }
+                if (i == 0 && j == 0) {
```

Comment:
Does this just stop the method from reassigning the MIN the first time the nested loops run?

Reply:
yes

Reply:
since i already assigned the MIN to be the element in the 2D array (arr[0][0])